frontend/src/lib/uPlotLib/utils/getXAxisScale.ts

```diff
@@ -36,5 +36,18 @@ export const getXAxisScale = (
 		maxTime = fallbackMax;
 	}
 
+	// As API response is adjusted to return values for  T - 1 min (end timestamp) due to which graph would not have the current timestamp data, we see the empty space in the graph for smaller timeframe. To accommodate this, we will be plotting the graph from (startTime  ->  endTime - 1 min).
```

Comment:
good to add a note why this is required

Reply:
_As API response is adjusted to return values for  T - 1 min (end timestamp) due to which graph would not have the current timestamp data, we see the empty space in the graph for smaller timeframe._ This part explains why it is required